src/components/QuranReader/SidebarNavigation/VerseList.tsx

```diff
@@ -37,14 +39,26 @@ const VerseList = () => {
     }
   }, [searchQuery, filteredVerseKeys]);
 
+  // Handle when user press `Enter` in input box
+  const handleVerseInputSubmit = (e) => {
+    e.preventDefault();
+    const selectedVerseKey = filteredVerseKeys.find((verseKey) => verseKey.endsWith(searchQuery));
```

Comment:
a qq, why cannot we use `filteredVerseKeys[0]` here instead of having to search again? this will have 2 advantages:

1. less code duplication
2. consistency with the way we are using `SurahList`.
3. Since `filteredVerseKeys` handles localized numbers, searching for localized verse key then pressing enter will work.